release: use goreleaser (#3619)
brimdata/zed: README.md

```
go install github.com/brimdata/zed/cmd/{zed,zq}@latest
```

> On a Mac, you can use `brew` to install `zq`:

Reviewer: ```suggestion
On macOS and Linux, you can use `brew` to install `zq`:
```
